src/api/app/models/package.rb

```diff
@@ -1242,9 +1250,16 @@ def self.verify_file!(pkg, name, content)
   def save_file(opt = {})
     content = '' # touch an empty file first
     content = opt[:file] if opt[:file]
+
```

Comment:
Moved it to ``verify_file!``! Only there it's causing issues because we call to_s on the UploadedFile object!